### ISSUE TITLE
Potted Torchflower Texture

### ISSUE THREAD

**Issue by MeltedMagma** [VixelCreates/VanillaEvolved#11]
Torchflower texture is bugged when put into a flower pot, you can see a 2x2 orange square from below but invisible from above. Definitely not like what the flower *should* look like.

**Comment by MeltedMagma** [comment]
Went at the issue again this morning and I altered the json file for the potted torchflower. I'm shocked I managed to get it to work (so quickly at that), but the issue seemed to be the fact that the numbers were off in the original file. I changed it to "0": "block/torchflower_model", and from the 2 that it was before and changed every 2 in that circumstance to a 0. I also copied the exact numbers in brackets regarding the flower from the torchflower.json file as there seemed to be major differences I wasn't able to figure out. This worked and now it looks normal. I uploaded a before/after photo and the updated json file. 
<img width="1135" alt="before and after" src="https://github.com/user-attachments/assets/3da58fed-4fd0-4f14-9cd5-32f237bbcff4">
[potted_torchflower.json](https://github.com/user-attachments/files/16613009/potted_torchflower.json)
